pkg/storage/segments/repairer.go

```diff
@@ -0,0 +1,157 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package segments
+
+import (
+	"context"
+
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/piecestore/psclient"
+	"storj.io/storj/pkg/pointerdb/pdbclient"
+	"storj.io/storj/pkg/storage/ec"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/pkg/utils"
+)
+
+// Repairer for segments
+type Repairer interface {
```

Comment:
The interface usually belongs to the place where it is being used, not what implements it. There are exceptions, but this allows for things to stay detached from their implementation.

Reply:
SGTM

Reply:
I found that the existing Repairer interface in the datarepair pkg was not used at all. So removed it and moved the Repair interface to the datarepair pkg - now with the name of "SegmentRepairer".